POP-CAR Added new content type Car

andresfernandezcsl opened the pull request:
Added new content type Car

## Description

- We added content type Car

## How to test

- [ ] [Create a intructuctions to review your PR , like go to /[anypath] and add a [anything]]


## A/C

- [ ] Add acceptance criteria for your PR like: - You should be able to add a new [anything].

## Extra Notes

- [ ] It's neccessary add a extra notes like another variables or observations to be considerer in this PR.

Commit:
Added new content type Car

Added new content type Car

jsteven97 commented:
Please @andresfernandezcsl try to be more accurate with the description 
<img width="1333" alt="image" src="https://github.com/jsteven97/pop-car/assets/48728210/aa71c18c-1b86-45c6-ad70-aa96d382efab">

jsteven97 commented:
Please @andresfernandezcsl con you name this fields labels in Spanish!
<img width="1646" alt="image" src="https://github.com/jsteven97/pop-car/assets/48728210/06b109e3-519d-42d0-a37f-f23f4a2217ef">
And this too in Spanish please
<img width="1759" alt="image" src="https://github.com/jsteven97/pop-car/assets/48728210/b08b136a-ce84-4c92-8f81-c40732b7a318">

jsteven97 commented:
Please can you create a list of color for this field 
<img width="860" alt="image" src="https://github.com/jsteven97/pop-car/assets/48728210/def71f07-91a7-4f22-be68-47dc3cc3a415">
If you need a list of color please let me know or use this 
**rojo, verde, azul, cian, magenta, amarillo, negro, blanco, gris, naranja, morado y rosa**

jsteven97 commented:
In this case is the same 
<img width="806" alt="image" src="https://github.com/jsteven97/pop-car/assets/48728210/cf5584bb-d633-4c57-8758-5a0ac3af6c51">

jsteven97 commented:
Please change this word 
<img width="412" alt="image" src="https://github.com/jsteven97/pop-car/assets/48728210/bbca0c43-f24b-4fca-8b89-f93b90374ea3">

- [ ] Fourth -> Cuatro
- [ ] Two -> Dos
- [ ] Fourth or more + Cuatro o más

Commit:
Car changes feedback

We did changes about translation in fields and content type, but we decided to keep color as a text field and not a list because there are a lot of colors and we don't want to restrict the user to a short list of colors.

jsteven97 commented:
I'm going to merge this content type, is perfect to explain some things for today training!
